rfcs/0095-empowering-new-scoped-module.md

```diff
@@ -0,0 +1,74 @@
+
+As in practices in other machine learning projects, there can be some levels of duplications or missing features compared with existing components (TorchFX TorchScript, TOSA and other MLIR graph IRs). Following the same practice in those related projects, as a team player in the community, one major principle in Apache is to empower communities by empowering optional components if they do not affect existing workflow. Empowering S0 through scoped modules brings a win-win situation for the community: it also brings in new aspirational members who are willing to collaborate and deliver the best for the community. This way, we keep ourselves up-to-date and grow stronger. On the other hand, failure to do so could result in community members getting discouraged and we lose valuable contributions and opportunities for us to grow in this rapidly growing area.
+
+The type of modules can include, but are not limited to:
+
+- IR dialects such as MLIR’s TOSA (while there are other graph IRs). TorchFX(while there is already TorchScript).
+- Vertical flows that leverage some of the dialects.
+- Backends/frontends.
+- Other types of modules introduced in a self-contained namespace.
+
+S0 changes would be contained in its namespace, with possible integrations also built inside its namespace. There can be follow-up steps (S1), such as making a dialect broadly accessible in existing compilation flows. Importantly, further S1/S2 level changes would require different RFC and longer deliberation for G0. The discussions on S1 also would serve as a way to allow the community to have a floor to talk about where broader areas of the project are going through a longer deliberation to maintain G0. Clearly identifying and empowering the S0 stage helps us to enable improvements quickly while bringing energies to the community, empowering a broader set of users, while not disrupting existing use cases.
+
+# Proposal: Empowering S0 Modules
+
+In this process RFC, We’d like to propose a process to encourage S0 modules and set expectations about what we anticipate in such inclusion.
+
+Note that this RFC focuses on the S0 stage. We propose the following guidelines to expedite to process while ensuring, quality and community support:
+
+- More than three PMC members endorse the S0-level proposal to ensure that there are enough eyes and shepherding in the module. The decision to establish a S0-level module needs to get majority support from PMC.
```

Comment:
Thank you for bringing this up. 2/3 majority decision normally applies to S1/S2 level decisions that have a bigger scope of impact. This is mainly considering different levels of open-mindedness people normally apply to decisions due to the difference in terms of scope of impact.

Reply:
So just to clarify this, if all but 3 PMC members voted against an S0 module, but 3 still endorsed it, would that vote succeed? Because I would expect that a module with minority support would have an extremely hard time getting PRs merged and could cause a lot of friction in the community.

Reply:
That's a great point. To prevent such cases, I change the text from "More than three PMC members" to "Majority of the PMC members". Thanks for your kind advice